### PR TITLE
read functionality

### PR DESCRIPTION
**Pull request by alvingithub1**
read functionality

### CODE REVIEW

File: src/pages/CatIndex.js [hunk 1]
```diff
@@ -1,9 +1,95 @@
 import React, { Component } from 'react'
+import { Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, Button } from 'reactstrap';
+import { NavLink } from 'react-router-dom'
+// import { Carousel, CarouselItem, CarouselControl, CarouselIndicators, CarouselCaption } from 'reactstrap';
+// import Carousel from 'react-multi-carousel';
 
 export class CatIndex extends Component {
+  // constructor(props) {
```
Comment (elysemontano) [inline src/pages/CatIndex.js]: There is a lot of commented-out code on this page.  It is best practice to remove any unused code.  It also looks like you have multiple exports on each file, usually, you will only want to have one export for the class.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Please remove unused code from CatIndex and also remove the extra exports on all of your pages and components.

**Comment by alvingithub1** [comment]
Removed commented code out and extra unused imports.

File: src/App.js [hunk 2]
```diff
@@ -8,20 +8,30 @@ import CatNew from './pages/CatNew'
 import CatShow from './pages/CatShow'
 import Home from './pages/Home'
 import NotFound from './pages/NotFound'
-import mockCats from './mockCats'
-
+import cats from './mockCats'
 import './App.css'
 
 export class App extends Component {
```
Comment (elysemontano) [inline src/App.js]: There are extra exports on all of your components and pages.  If you export default at the bottom of the component, you do not need the export before declaring the class. 
```suggestion
class App extends Component {
```

**Review by elysemontano — CHANGES_REQUESTED** [review]
I left a comment in App.js that shows a little more clarity on what I meant by extra exports.  Please check all of your components and pages and remove the extra exports.

**Comment by alvingithub1** [comment]
Removed extra exports as requested.

**Review by elysemontano — APPROVED** [review]
Looks good!

**Pull request merged**